docs/tools/kubernetes.md

```diff
@@ -2,114 +2,39 @@
 
 ## Get started
 
 Take your first steps by configuring the Aiven Operator and deploying a PostgreSQL®
 database.
 
-### Requirements
-
-- You have access to a Kubernetes cluster where you can run the operator.
-- You have an Aiven account. If you don't have one yet,
-  [sign up for free](https://console.aiven.io/signup).
-- To use the operator locally, we recommend using [**kind**](https://kind.sigs.k8s.io/).
-  See [the installation guide](https://kind.sigs.k8s.io/docs/user/quick-start/#installation).
-- To install the operator, we recommend [**Helm**](https://helm.sh/).
-  See the [installation guide](https://helm.sh/docs/intro/install/).
-
-### Install the Kubernetes® operator
-
-1. Optional: To manage the webhook TLS certificates used by our operator,
-   install the `cert-manager`:
-
-   ```bash
-   kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
-   ```
-
-   :::tip
-   You can also use the operator without `cert-manager` and the admission
-   webhooks. To do so, skip this step and go to step 3 to install Helm.
-   :::
-
-   :::note
-   - For GKE version >= 1.21 VPC users, add a firewall rule to explicitly allow
-     ingress to port 9443 for admission webhook. See
-     [the Google Cloud docs](https://cloud.google.com/kubernetes-engine/docs/how-to/private-clusters#add_firewall_rules).
-
-   - If you are running a GKE _Autopilot_ cluster in Google Cloud Platform,
-     use GKE version >=1.21 and install `cert-manager`
-     into the `cert-manager` namespace, as per [this GitHub issue
-     comment](https://github.com/cert-manager/cert-manager/issues/3717#issuecomment-975031637)
-   :::
-
-1. Verify the `cert-manager` installation by checking if the pods are up
-   and running:
-
-   ```bash
-   kubectl get pod -n cert-manager
-   ```
-
-   Ensure the pod's status is `Running`.
-
-1. Add the [Aiven Helm chart
-   repository](https://github.com/aiven/aiven-charts/) and update your
-   local Helm information:
-
-   ```bash
-   helm repo add aiven https://aiven.github.io/aiven-charts
-   helm repo update
-   ```
-
-1. Install the CRD and the operator itself:
-
-   ```bash
-   helm install aiven-operator-crds aiven/aiven-operator-crds
-   helm install aiven-operator aiven/aiven-operator
-   ```
-
-   :::tip
-   To disable the admission webhooks, run:
-   ```
-   helm install aiven-operator aiven/aiven-operator --set webhooks.enabled=false
-   ```
-   :::
-
-1. Verify the installation by making sure the operator pod is running:
+## Prerequisites
 
-   ```bash
-   kubectl get pod -l app.kubernetes.io/name=aiven-operator
-   ```
-
-Your pod's status should be `Running`.
+- [Sign up for Aiven](https://console.aiven.io/signup).
+- [Install the Aiven Operator](https://aiven.github.io/aiven-operator/installation/helm.html).
+- Have admin access to a Kubernetes cluster where you can run the operator.
 
 ### Authenticating
```

Comment:
How about using `Authenticate` (no `-ing`) to stay consistent with the other section titles using the imperative mood?